docs/pages/frameworks/react.md

```diff
@@ -83,6 +83,25 @@ function MyComponent() {
 export default MyComponent;
 ```
 
+You can also import the event type for use in your callbacks, shown below.
+
+```tsx
+import { useState } from 'react';
```

Comment:
Looks like we're missing an import here.

```suggestion
import { useCallback, useState } from 'react';
```

Reply:
good catch - is there a test or lint i can use to check the docs suggestions are valid?

Reply:
Not really. We have to run them manually with the way it's set up right now. Clicking the CodePen button with React selected is a manual workaround.